cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -219,14 +220,14 @@ def update(self, updates, cancel=False, reset=False):
         if reset:
             view.reset_button.click()
             flash_message = 'All changes have been reset'
-            flash_view = ServerCollectLogsEditView
```

Comment:
For the record (merging this), I don't quite like this pattern of setting these view attributes for use in flash message validation.  We prefer to have these flash message checks in the CRUD test for related collection/entity - and the flash message is available by using a generic `BaseLoggedInView`. 

Ultimately standardizing that pattern is an enhancement on the framework overall.

Reply:
I entirely agree